packages/atclient/src/atnotification.c

```diff
@@ -84,340 +89,165 @@ void atclient_atnotification_free(atclient_atnotification *notification) {
   }
 }
 
-int atclient_atnotification_from_json_str(atclient_atnotification *notification, const char *json_str) {
```

Comment:
The diff in this file is really messed up because it was misformatted

I've taken the json-library specific functions and moved them to the bottom of the file